src/engine/command.cpp

```diff
@@ -1028,8 +1028,32 @@ static inline void skipcomments(const char *&p)
     for(;;)
     {
         p += strspn(p, " \t\r");
-        if(p[0]!='/' || p[1]!='/') break;
-        p += strcspn(p, "\n\0");
+        if(p[0] == '/' && p[1] == '/')
```

Comment:
I think we should document that this block is supposed to handle single-line comments, like the others handle multi-line ones. Also, a blank line above this one will increase readability.